LayoutTests/imported/w3c/web-platform-tests/html/webappapis/scripting/events/event-handler-all-global-events-expected.txt

```diff
@@ -58,7 +58,7 @@ FAIL oncontextlost: must be on the appropriate locations for GlobalEventHandlers
 FAIL oncontextlost: the default value must be null assert_equals: The default value of the property is null for a Window instance expected (object) null but got (undefined) undefined
 FAIL oncontextlost: the content attribute must be compiled into a function as the corresponding property assert_equals: The oncontextlost property must be a function expected "function" but got "undefined"
 FAIL oncontextlost: the content attribute must execute when an event is dispatched assert_true: Dispatching an event must run the code expected true got undefined
-PASS oncontextlost: dispatching an Event at a <meta> element must trigger element.oncontextlost
+FAIL oncontextlost: dispatching an Event at a <meta> element must trigger element.oncontextlost assert_true: expected true got false
```

Comment:
Is this OK?

Reply:
Yeah, this is a result of me making that test more strict. Before this change it would pass if the event listener didn't run.